content/administer-tugboat-crew/change-permissions.md

```diff
@@ -10,8 +10,11 @@ weight: 3
 2. Select the project where you want to change user permissions.
 3. Click the {{% ui-text %}}Project Settings{{% /ui-text %}} link to the right of the project's title.
 4. In the {{% ui-text %}}Manage Users{{% /ui-text %}} section, look for the user whose permissions you want to change,
-   and select the appropriate [user type](#user-permission-levels-explained) from the
-   {{% ui-text %}}Access{{% /ui-text %}} drop-down.
+   and select the appropriate [user type](../user-admin/) from the {{% ui-text %}}Access{{% /ui-text %}} drop-down.
+
+{{% notice info %}} You can't downgrade an [Owner user's permissions](../user-admin/#owner-permissions) without first
```

Comment:
This is true. But, in order to downgrade an owner, you already have to be an owner yourself. You can't remove yourself, only another owner can. The way this is worded here kind of implies that there needs to be two owners, you, and another user that you give owner permissions to, before you can downgrade another owner.

That feels rambly. I hope it makes sense?

Reply:
I reworded this, but am second-guessing whether this verbiage is needed. Thoughts?

Reply:
My instinct is that it's probably not necessary to have this callout. It's difficult to clearly explain, and I think the interface makes it fairly obvious.